Extension/src/common.ts

```diff
@@ -123,10 +124,24 @@ export function resolveVariables(input: string): string {
         return "";
     }
 
-    // Replace environment variables. (support both ${env:VAR} and ${VAR} syntax)
-    let regexp: RegExp = /\$\{(env:|env.)?(.*?)\}/g;
-    let ret: string = input.replace(regexp, (match: string, ignored: string, name: string) => {
-        let newValue: string = process.env[name];
+    // Replace environment and configuration variables.
+    let regexp: RegExp = /\$\{((env|config)(.|:))?(.*?)\}/g;
+    let ret: string = input.replace(regexp, (match: string, ignored1: string, varType: string, ignored2: string, name: string) => {
+        // Historically, if the variable didn't have anything before the "." or ":"
+        // it was assumed to be an environment variable
+        if (varType === undefined) { varType = "env"; }
```

Comment:
>varType = "env"; } [](start = 37, length = 18)

I believe our linting rules require the body of the if to be on a new line.

Reply:
I ran TSLint and got "vscode-tslint: Status is OK" (It did find some other lint issues though).
Happy to move it to new line, though!